consensus_pbft.go

```diff
@@ -74,9 +75,12 @@ type Pbft struct {
 	// Output logger
 	logger Logger
 
-	// Config is the configuration of the consensus
+	// config is the configuration of the consensus
 	config *Config
 
+	// votingMetadata encapsulates voting votingMetadata (such as quorum size calculation, max faulty nodes etc.)
+	votingMetadata VotingMetadata
```

Comment:
This is not a parameter of `Pbft` but of the specific backend instance.

Reply:
In that case, `Backend` implementation (e.g. `fsm`) provides `VotingMetadata` object to the consensus algorithm is that correct?